book.adoc

```diff
@@ -26,6 +49,10 @@ Antoine Sabot-Durand <antoine@sabot-durand.net>
 === Getting and installing Tools used in this book
 === CDI website and resources
 === Choosing your implementation
+=== Using CDI with Java SE
```

Comment:
Idem here. We still have to think about it but I think the example application will target Java EE to avoid dealing with integration stuff in the fist place (integration means "extension" very often).